tests-functional/tests/test_wakuext_message_reactions.py

```diff
@@ -106,3 +107,71 @@ def test_one_to_one_message_reactions(self, backend_new_profile, waku_light_clie
                     item["emoji"] == "🙂",
                 )
             )
+
+    @pytest.mark.parametrize("waku_light_client", [False], indirect=True, ids=["wakuV2LightClient_False"])
+    def test_limit_of_20_reactions(self, backend_new_profile, waku_light_client):
+        """Test that you cannot send more than 20 message reactions on a single message"""
+        # Initialize two backends (sender and receiver) for this test
+        self.sender = backend_new_profile("sender", waku_light_client=waku_light_client)
+        self.receiver = backend_new_profile("receiver", waku_light_client=waku_light_client)
+
+        self.make_contacts(self.sender, self.receiver)
+        response = self.sender.wakuext_service.send_one_to_one_message(self.receiver.public_key, "test_message")
+        message = self.get_message_by_content_type(response, content_type=MessageContentType.TEXT_PLAIN.value)[0]
+        message_id, sender_chat_id = message["id"], message["chatId"]
+        receiver_chat_id = self.receiver.wakuext_service.rpc_request(method="chats")[0]["id"]
+
+        # Send 20 emojis
+        emojis = ["😀", "🤓", "😄", "😁", "😆", "😅", "🤣", "😂", "🥹", "🙂", "🙃", "😉", "😊", "😇", "🥰", "😍", "🤩", "😘", "😗", "☺️"]
+        last_emoji_id = None
+        for emoji in emojis:
+            response = self.sender.wakuext_service.rpc_request(method="sendEmojiReactionV2", params=[sender_chat_id, message_id, emoji])
+            assert response["emojiReactions"][0]["emoji"] == emoji
+            last_emoji_id = response["emojiReactions"][0]["id"]
+
+        # The 21st emoji sent should fail
+        try:
+            _ = self.sender.wakuext_service.rpc_request(method="sendEmojiReactionV2", params=[sender_chat_id, message_id, "😋"])
+            assert False, "Expected an error when sending the 21st emoji reaction, but the request succeeded."
+        except ApiResponseError as err:
+            assert "too many emoji reactions for message" in str(err)
```

Comment:
We use `with pytest.raises` for the case when an error is expected. Then you won't need to `assert False`.
(you can find examples in the tests by searching "pytest.raises")